### ISSUE TITLE
Allow app to be limited to one channel

### ISSUE THREAD

**Issue by wilsonpage** [wilsonpage/songwhip-slack#1]
>Love your service. I want to install your Slack app in my work Slack, but we have strict compliance and security requirements. Is there any way to limit your app to one channel? Otherwise we can't add it.

**Comment by wilsonpage** [comment]
> Nathan Leiby @nathanleiby
> 1. "Man would be nice if it inlined the links but alas" -- might you inline links as replies? (or make it configureable?)
> 2. possible to add filtering on which channels the integration is enabled for?

**Comment by cnaurois** [comment]
Definitely a good idea to restrain to some channels if possible.
We prefer to limit activity to #nowplaying or #jukebox

**Comment by avkvirtru** [comment]
Yep, I work for [Virtru](https://www.virtru.com), a privacy & security startup. We have a #radio-virtru channel and, given our business, strict screening for any third-party apps that can read our data in Slack.
